e2e/app/testdata/TestWriteGethConfigTOML.golden

```diff
@@ -27,5 +27,7 @@ BootstrapNodes = ["enode://3a514176466fa815ed481ffad09110a2d344f6c9b78c1d14afc35
 TrustedNodes = ["enode://3a514176466fa815ed481ffad09110a2d344f6c9b78c1d14afc351c3a51be33d8072e77939dc03ba44790779b7a1025baf3003f6732430e20cd9b76d953391b3@127.0.0.1:1","enode://3a514176466fa815ed481ffad09110a2d344f6c9b78c1d14afc351c3a51be33d8072e77939dc03ba44790779b7a1025baf3003f6732430e20cd9b76d953391b3@127.0.0.2:2"]
 ListenAddr = "0.0.0.0:30303"
 
-# TODO(corver): Enable prometheus metrics.
-# [Metrics]
+[Metrics]
+Enabled = true
```

Comment:
Do we need the `[Metrics]` section in the config if we have the `--metrics` flag in the command?